Compilation fails on SAMD51 board without a Serial1 declaration

### Operating System

MacOS

### IDE version

Arduino 1.8.15

### Board

Adafruit Monster M4sk

### BSP version

1.7.3

### TinyUSB Library version

1.3.0 via Arduino Library Manager

### Sketch

Adafruit_Learning_System_Guides/M4_Eyes

### What happened ?

Project compilation fails with the following error:
/Users/pburgess/Documents/Arduino/libraries/Adafruit_TinyUSB_Library/src/arduino/ports/samd/Adafruit_TinyUSB_samd.cpp:65:3: error: 'Serial1' was not declared in this scope; did you mean 'Serial'?

**This board (Monster M4sk) does not have a Serial1 declaration in ArduinoCore_samd.**

Workaround is to edit tusb_config_samd.h and change line 49 to:
`#define CFG_TUSB_DEBUG 0`
Project then builds, uploads and runs successfully.

Perhaps this should be the default case when headed for the Arduino Library Manager? Or, if debug 2 is the default and intended case, maybe the board definition should get a Serial1 added, in which case I’ll check for any other SAMD51 boards without one.

### How to reproduce ?

Build project described above

### Debug Log

_No response_

### Screenshots

_No response_

--- a/changelog.md
+++ b/changelog.md
@@ -1,5 +1,23 @@
 # Adafruit TinyUSB Arduino Library Changelog
 
+## 1.3.2 - 2021.07.07
+
+- revert CFG_TUSB_DEBUG = 0 on SAMD port
+
+## 1.3.1 - 2021.07.06
+
+- Fix warning with ci build for rp2040 core
+- Fix example build for esp32s2
+- Use ARDUINO_NRF52_ADAFRUIT instead of ARDUINO_ARCH_NRF52
+
+## 1.3.0 - 2021.06.29
+
+- Move tusb_config for each ports into library to make it more portable
+- Adding support for ESP32S2 (still need PR to be merged from esp32-arduino)
+- Update CDC to have instance and getInstanceCount()
+- Allow USB_VID/PID to be declared in variant pins_arduino.h
+- Use bug report form template
+
 ## 1.1.0 - 2021.06.21
 
 - Add support for multiple CDC ports (need to modify tusb_config.h)

--- a/library.properties
+++ b/library.properties
@@ -1,5 +1,5 @@
 name=Adafruit TinyUSB Library
-version=1.3.1
+version=1.3.2
 author=Adafruit
 maintainer=Adafruit <info@adafruit.com>
 sentence=TinyUSB library for Arduino

--- a/src/arduino/ports/samd/tusb_config_samd.h
+++ b/src/arduino/ports/samd/tusb_config_samd.h
@@ -46,7 +46,7 @@ extern "C" {
 
 #define CFG_TUSB_OS OPT_OS_NONE
 
-#define CFG_TUSB_DEBUG 2
+#define CFG_TUSB_DEBUG 0
 #if CFG_TUSB_DEBUG
 #define CFG_TUSB_DEBUG_PRINTF serial1_printf
 #endif

